packages/markdown/remark/src/index.ts

```diff
@@ -86,8 +98,8 @@ export async function renderMarkdown(
 	try {
 		const vfile = await parser
 			.use([rehypeCollectHeaders])
-			.use(rehypeStringify, { allowDangerousHtml: true })
-			.process(content);
+			.use(rehypeStringify, { allowDangerousHtml: true, allowParseErrors: true })
```

Comment:
well that's a bit of a scary flag name 😆

Does this allow MDX syntax to get through without being thrown as bad markdown syntax?